trulens_eval/trulens_eval/feedback/provider/endpoint/base.py

```diff
@@ -777,7 +786,8 @@ def __init__(
             # Already created with SingletonPerName mechanism
             return
 
-        assert error_prob + freeze_prob + overloaded_prob + loading_prob <= 1.0, "Probabilites should not exceed 1.0 ."
+        assert error_prob + freeze_prob + overloaded_prob + \
+            loading_prob <= 1.0, "Probabilites should not exceed 1.0 ."
```

Comment:
Why do we have all the format changes? Did we change any formatting tool?

Reply:
previously there's not a unified formatter used in Trulens AFAIK. 
Now we do have `yapf` and I'll clean this PR up soon